README.md

```diff
@@ -36,4 +36,68 @@ To run all microservices docker containers, simply run the following command:
 
 To stop all containers, run the following:
 
-` docker-compose stop `
\ No newline at end of file
+` docker-compose stop `
+
+## Microservice Database Creation
+
+### Prerequisites
+* A MySQL 5.6.x server
```

Comment:
A fair point; I suppose we don't have to call out a specific version of MySQL.  Locally I have MySQL 5.7.17, and this script ran fine against it.  I called this out because that's the version our DEV & QA (and presumably load test) servers are at, so that's the version we tested against.